src/composers/field.tsx

```diff
@@ -59,10 +62,11 @@ export function Field<Props>(WrappedComponent: IncomingField<Props>,
         }
 
         public render() {
+            const fieldState = this.props.form.getFieldState(this.getFieldName());
```

Comment:
This might be the textbook use-case for `forceUpdate`, since our state is in another place.. so we can just remove local state and call the `getFieldState` in the render function https://facebook.github.io/react/docs/react-component.html#forceupdate